mcnp5/patch/dagmc.patch.5.1.60

```diff
@@ -353,7 +353,7 @@ diff -rN '--unified=0' mcnp_vendor/Source/src/dagmc_mod.F90 mcnp_dagmc/Source/sr
 +
 +
 +     ! parse metadata and write input cards
-+     call dagmcwritemcnp(gcad//char(0),lcad//char(0), len_trim(lcad))
++     call dagmcwritemcnp(lcad//char(0), len_trim(lcad))

```

Comment:
Hmmm... the CAE repo is authoritative for MCNP5.  We need to make sure these are all properly updated.